app/dts/bindings/behaviors/zmk,behavior-macro.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Macro Behavior
+
+compatible: "zmk,behavior-macro"
+
+include: zero_param.yaml
+
+properties:
+  bindings:
+    type: phandle-array
+    required: true
+  wait-ms:
```

Comment:
Should we put help text on these to clarify what the tap time and wait time do?

Reply:
Added `descriptions` to each of these properties. Thoughts on the specific descriptions?

Reply:
You might explicitly specify "time in milliseconds" in case the -ms suffix isn't obvious, but otherwise those seem understandable to me.

Reply:
Added the units in the descriptions.